app/views/partials/tracking_events/_view_item_content.html.haml

```diff
@@ -0,0 +1,15 @@
+:javascript
+  fbq('track', 'ViewContent', {
+    content_name: '#{item.name}',
+    content_ids: ['#{item.id}'],
+    content_type: 'product',
+    value: '#{item.price}',
+    currency: 'TWD'
+  });
+
+  ga('ec:addImpression', {
+    'id': '#{item.id}',
+    'name': '#{item.name}',
+    'price': '#{item.price}'
+  });
+  ga('ec:setAction', 'detail');
```

Comment:
Note: Ecommerce data can only be sent with an existing hit, for example a pageview or event. If you use ecommerce commands but do not send any hits, or the hit is sent before the ecommerce command then the ecommerce data will not be sent.